discourse-dark.user.css

```diff
@@ -1182,11 +1182,13 @@ domain("discourse.huel.com"),
 domain("discuss.huel.com"),
 domain("discuss.ocaml.org"),
 domain("forum.f-droid.org"),
+domain("community.roonlabs.com"),
```

Comment:
```suggestion
```
This is not needed since you already added `regexp("^https://community\\.(roonlabs|`

Reply:
I'm finding that the logo inversion doesn't work without this.

Reply:
> I'm finding that the logo inversion doesn't work without this.

What we normally do is override such site specific on its own `@-moz-document` section, so as to avoid making changes to other sites that probably would conflict.

See https://github.com/StylishThemes/Discourse-Dark/commit/655f27c9bc44ad7a698406ef6edbee9ee4ef7f32 more specifically 

https://github.com/StylishThemes/Discourse-Dark/commit/655f27c9bc44ad7a698406ef6edbee9ee4ef7f32#diff-957b388e3653ee43cb382b14ae4b7741R1180-R1192

In future if you have fixes for roon just add them to this section, unless its a global change that applies to most of the supported sites, will make maintenance a lot easier and simpler.

Thanks, hope 1.6.0 is working for you.